packages/suite/src/views/settings/SettingsDevice/WipeDevice.tsx

```diff
@@ -9,15 +11,30 @@ interface WipeDeviceProps {
 }
 
 export const WipeDevice = ({ isDeviceLocked }: WipeDeviceProps) => {
+    const { device } = useDevice();
     const dispatch = useDispatch();
 
+    const isBootloaderMode = isDeviceInBootloaderMode(device);
+
     const handleClick = () => dispatch(openModal({ type: 'wipe-device' }));
 
+    const headingTranslation = isBootloaderMode
```

Comment:
It should be the other way round: Factory reset in bootloader, wipe in normal. Otherwise OK.